### ISSUE TITLE
Shipping Box based on shipping classes

### ISSUE THREAD

**Issue by Red-Cross-Knight** [flagshipcompany/flagship-for-woocommerce#55]
Background: We ship dry and frozen goods, frozen goods require a insulated box so higher box cost. 

Under Parcel / Packaging, add column for shipping class for each Package Box.

**Comment by patrick-fls** [comment]
Hi @Red-Cross-Knight,

Thanks for the suggestion! We're definitely going to look at it for the next release of the plugin.

**Comment by Red-Cross-Knight** [comment]
Any news on this one?  Due to covid-19 our shipments have tripled and this would be very helpful.

**Comment by luis709394** [comment]
Hi,

We already added this new feature in the release of 2.0.14. Now if you go
to the plugin settings page, you will see a new column for box markup in
the packing boxes (see the screenshot attached).

**Comment by Red-Cross-Knight** [comment]
Yes, that works well, currently using that, however it is not dependent on shipping class just on all boxes (unless I'm missing something?)  (running Version 2.0.15)

**Comment by luis709394** [comment]
Sorry, I think I did not read the ticket carefully and gave you the incorrect answer last time. We added a new column for the extra markup of each box last year. But that extra markup will be applied to the box every time regardless of the shipping class of the products. In your suggestion, there should be a column for each box for the shipping class of products the box is used for. For example, if in the shipping class of a box it says "frozen", the box will only be used for products belonging to the "frozen" shipping class. And if you put extra markup for that box, the shipping of those "frozen" products will require extra fees.  Is this correct?

If that is the case, I need to discuss with the rest of our team to decide the implementation of this new feature. I will get back to you soon.


Thanks!
